Gemfile

```diff
@@ -79,6 +79,9 @@ gem "stimulus-rails"
 # User-Resource Permissions and Scoping
 gem "pundit"
 
+gem "sentry-rails"
+gem "sentry-ruby"
```

Comment:
Do we need to explicitly add `sentry-ruby`? I think it's already a dependency for `sentry-rails`. :)